worker/pom.xml

```diff
@@ -98,6 +98,11 @@
             <version>${newrelic.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcpkix-jdk15on</artifactId>
+            <version>${bounceycastle.version}</version>
+        </dependency>
```

Comment:
Also, we _might_ have Apache Coded already on the classpath, in which case you could just use `org.apache.commons.codec.binary.Hex` for Hex-ing